extra/simplesamlphp.spec

```diff
@@ -0,0 +1,94 @@
+%define name      simplesamlphp
+%define summary   SAML IDP/SP written i PHP
+%define version   1.13.2
+%define release   10
+%define license   LGPL 2.1
+%define group     Networking/WWW
+%define source    %{name}-%version.tar.gz
+%define url       https://simplesamlphp.org
+%define vendor    Feide RND
+%define buildroot %{_tmppath}/%{name}-root
+%define _prefix   /var/lib/
+
+Name:      %{name}
+Summary:   %{summary}
+Version:   %{version}
+Release:   %{release}
+License:   %{license}
+Group:     %{group}
+Source0:   %{source}
+BuildArch: noarch
+Requires:  httpd, mod_ssl, mod_php, php-ldap, php-mcrypt, php-pecl-krb5, php-xml
```

Comment:
I don't think you should require php-pecl-krb5. I doubt it's used for anything unless you are using the negotiate module, which is just a marginal usage of SSP.